### ISSUE TITLE
insert after pattern in "modify" instead of replacing pattern

### ISSUE THREAD

**Issue by macrozone** [amwmedia/plop#102]
I have a generator that appends items based on a template to a list:

```
// list-template

/* 💬 LIST */

// list-item-template.js ( e.g. a reducer)

$1
[{{actionName}}]: (state, action) => ({
  ...state,
  // your property: action.payload
})

```

Then i can append items to LIST with this:

```
{
        type: 'modify',
        path,
        pattern: /(\/\* 💬 LIST \*\/)/gi,
        templateFile: 'templates/list-item-template.js',
      },
```

This works, but it's easy to forget the `$1` in the `list-item-template`. It's needed, otherwise the "magic" comment `/* 💬 LIST */` will be removed and this modification only works once. 

Maybe we need another type `append` which looks for a pattern and appends `template` after the pattern. Or is there a more elegant solution to that?

**Comment by macrozone** [comment]
ok i created a PR for that: https://github.com/amwmedia/node-plop/pull/56

**Comment by amwmedia** [comment]
this has been merged and slated for release in v2.0
